dmaws/commands/exportdata.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import click
+
+from ..cli import cli_command
+from ..stacks import StackPlan
+from ..rds import RDS, RDSPostgresClient
+
+
+@cli_command('exportdata', max_apps=0)
+@click.option('--export-path', default='.',
+              help="Path to write export files.")
+def exportdata_cmd(ctx, export_path):
+    plan = StackPlan.from_ctx(ctx, apps=['database_dev_access'])
+
+    if not plan.create(create_dependencies=False):
```

Comment:
To the first question; yes. Although that's a good point, it would be good to handle that in the RDS client, I'll take a look at it.